doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_arcgis_featuretable_sink.md

```diff
@@ -40,29 +42,75 @@ Argis feature tables must be provisioned before sending entities.
 #### <a name="section1.2.1"></a>ArcGis databases naming conventions
 Each entity type needs an url and an unique field to be persisted into the feature table.
 
-NGSIArcgisFeatureTableSink composes each table's url with entitie's `service` and `service path`, to provide multiple tables access. 
+NGSIArcgisFeatureTableSink composes each table's url with entitie's `service` and `service path`, to provide multiple tables access. The final url is compounds by `cygnus-ngsi.sinks.arcgis-sink.arcgis_service_url`+`fiware-service`+`fiware-servicepath`. 
 
-Unique field is provided to allow `NGSIArcgisFeatureTableSink` to update existant entities. NGSI `entity type` will be used as unique field name.
+Unique field is provided to allow `NGSIArcgisFeatureTableSink` to update existant entities. NGSI `entity type` will be used as unique field name. This means that a feature named `type` in the Feature Table cannot be filled in by the sink. If Feature Table needs to persist the value of entity type it has to be in a feature different than `type`.
 
-All this parameters, can be customized using Cygnus mapping capabilities.
+All this parameters, can be customized using Cygnus mapping capabilities or Context Broker custom notifications.
+
+##### Using Cygnus
 
 Let's see an example:	
 
-##### Agent.conf file:
+###### Agent.conf file:
 
 	agent.arcgis-sink.arcgis_service_url = https://arcgis.com/{hash}/arcgis/rest/services
-##### Entity data:
+###### Entity data:
 
 	service = vehicles
 	service-path = /4wheels
 	entity-type = car
-##### result
+###### result
 
 	Feature table url: https://arcgis.com/{hash}/arcgis/rest/services/vehicles/4wheels
 	Table's unique field: car
 
 [Top](#top)
 
+#### Using Context Broker custom notifications
+
+If a service path has different entitiy types, following the previous example `car` and `van`, they will need to persist in different Feature Layers, to avoid the limitation of one "fiware-servicepath" can only persist in a unique feature table, it can be used [CB custom notifications](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#custom-notifications) in which we can map a different value for `fiware-service` and `fiware-servicepath` headers parameters. 
+
+###### Entity data:
+
+	service = vehicles
+	service-path = /4wheels
+	entity-type = van
+
+For example, if the Feature table for "van" is `https://arcgis.com/{hash}/arcgis/rest/services/vehicles/van`,  the custom notif could be:
```

Comment:
I'd suggest to include the full JSON of the subscription, including the `subject` object, so it can be seen how `"type": "van" is used in this case).

Reply:
In addition, review the whole document to change:

* van -> Van
* car -> Car

and any other type so we use the current cannonical way of naming entity types (i.e. with first capital letter).